payments-core-testing/src/main/java/com/stripe/android/testing/AbsFakeStripeRepository.kt

```diff
@@ -156,7 +156,6 @@ abstract class AbsFakeStripeRepository : StripeRepository {
     @Throws(APIException::class)
```

Comment:
Nit: Not introduced here, but can you also get rid of the incorrect `Throws` annotations?